x-pack/legacy/plugins/canvas/public/components/arg_form/arg_simple_form.tsx

```diff
@@ -4,22 +4,34 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { ReactNode, MouseEventHandler } from 'react';
 import PropTypes from 'prop-types';
 import { EuiButtonIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
-import { TooltipIcon } from '../tooltip_icon';
+import { TooltipIcon, IconType } from '../tooltip_icon';
+
+import { ComponentStrings } from '../../../i18n';
+
+const { ArgFormArgSimpleForm: strings } = ComponentStrings;
+
+interface Props {
+  children?: ReactNode;
+  required?: boolean;
+  valueMissing?: boolean;
+
```

Comment:
Blank line? Trying to pad your line counts, I see.  🤣

Reply:
vertical whitespace isn't a crime!!!!

Reply:
although it doesn't really make sense here haha oops